src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -737,7 +737,7 @@
     <value>Command Palette</value>
   </data>
   <data name="AIChatMenuItem" xml:space="preserve">
-    <value>Terminal Chat (experimental)</value>
+    <value>Terminal Chat</value>

```

Comment:
>Terminal Chat [](http://example.com/codeflow?start=11&length=13)

Can you remind me the rationale for removing it from the dropdown? Was it to save the horizontal real estate?

Reply:
From @nguyen-dows:

> I personally don’t think that “Experimental” should be marked on dropdown menu items especially when you’ll already see “Terminal Chat (Experimental)” after opening up the chat UI

Reply:
^ Horizontal real estate was also a factor in this decision in excluding that word :)

Reply:
Makes sense- thanks, guys!